src/main/java/com/lothrazar/storagenetwork/capability/CapabilityConnectableLink.java

```diff
@@ -289,7 +289,11 @@ public void addToStackProviderBatch(Batch<StackProvider> availableItems) {
     if (inventoryFace == null) {
       return;
     }
-    DimPos inventoryPos = connectable.getPos().offset(inventoryFace);
+    DimPos connectablePos = connectable.getPos();
+    if (connectablePos == null) {
```

Comment:
perfect looks like this fixes https://github.com/Lothrazar/Storage-Network/issues/513